src/pages/Transfer/Transfer.scss

```diff
@@ -0,0 +1,34 @@
+@import "../../styles/_variables.scss";
+
+.transfer {
+    background-color: $background-color;
+    color: $text-color;
+
+    display: grid;
+    grid-template-areas: "flag currency armount rate button";
+
+    .transfer__flag {
+        grid-area: "flag";
+    }
+
+    .transfer__currency {
+        grid-area: "currency";
+    }
+
+    .transfer__amount {
+        grid-area: "amount";
+    }
+
+    .transfer__rate {
+        grid-area: "rate"
+    }
+
+    .transfer__button {
+        grid-area: "button";
+    }
+}
+
+.transfer__active {
```

Comment:
could this not be in initial nested sass block?

Reply:
Not easily as far as I can tell, since it's running on the outmost div, and nesting it would try to make it search inside the div, instead of for the div itself.